<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">

  <property name="geometry">
   <rect><x>0</x><y>0</y><width>750</width><height>520</height></rect>
  </property>

  <property name="windowTitle">
   <string>Transaction History</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">

    <!-- Title -->
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text"><string>Your Transaction History</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
      <property name="font">
       <font><bold>true</bold><pointsize>18</pointsize></font>
      </property>
     </widget>
    </item>

    <!-- NEW: Select Account Label -->
    <item>
     <widget class="QLabel" name="labelSelect">
      <property name="text"><string>Select Account:</string></property>
      <property name="font">
       <font><bold>true</bold><pointsize>12</pointsize></font>
      </property>
     </widget>
    </item>

    <!-- NEW: Account Combo -->
    <item>
     <widget class="QComboBox" name="accountCombo">
      <property name="minimumHeight"><number>35</number></property>
     </widget>
    </item>

    <!-- Table for transactions -->
    <item>
     <widget class="QTableWidget" name="tableTransactions"/>
    </item>

    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text"><string>Back</string></property>
      <property name="minimumHeight"><number>40</number></property>
     </widget>
    </item>

   </layout>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
